src/control/Control.Layers.js

```diff
@@ -162,6 +166,9 @@ export const Layers = Control.extend({
 	// @method expand(): this
 	// Expand the control container if collapsed.
 	expand() {
+		if (this.collapseDelayTimeout) {

```

Comment:
```suggestion
		if (this._collapseDelayTimeout) {
```
please make the variable private by adding `_`